includes/TripalFields/local__go_results/local__go_results_formatter.inc

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * @class
+ * Purpose:
+ *
+ * Display:
+ * Configuration:
+ */
+class local__go_results_formatter extends ChadoFieldFormatter {
+
+  // The default label for this field.
+  public static $default_label = 'Go results';
+
+  // The list of field types for which this formatter is appropriate.
+  public static $field_types = array('local__go_results');
+
+  // The list of default settings for this formatter.
+  public static $default_settings = array(
+    'setting1' => 'default_value',
+  );
+
+   /**
+    * @see ChadoFieldFormatter::settingsForm()
+    *
+    **/
+
+  public function settingsForm($view_mode, $form, &$form_state) {
+
+  }
+
+    /**
+    * @see ChadoFieldFormatter::View()
+    *
+    **/
+
+  public function view(&$element, $entity_type, $entity, $langcode, $items, $display) {
+
+    // Get the settings
+    $settings = $display['settings'];
+
+    $organism_id = $entity->chado_record->organism_id;
+
+    // Check for permission to see Analysis entity
+
+    $query = db_select('public.tripal_bundle', 'tb');
+    $query->join('tripal_term', 'tt', 'tt.id = tb.term_id');
+    $bundle_id = $query->fields('tb', ['name'])
+      ->condition('tt.name', 'Analysis')
+      ->execute()->fetchField();
+
```

Comment:
But what if the admin is using a custom analysis type?

Reply:
Well, it works with files loaded with the Interpro load 

![image](https://user-images.githubusercontent.com/17642511/51938634-e3691400-240d-11e9-8aa9-41b280c7a8d6.png)

I'm not sure about other custom analysis types, it would depend on the loader itself I think?

Reply:
>Well, it works with files loaded with the Interpro load

awesome!  sorry im still loading GO terms on my site so i can test this :D

In that case, you have to consider cases where people loaded the IPR terms attaching it to the `InterPro Results` bundle, not just the `Analysis` bundle. 

Also I don't know that its necessary to check for the bundle view permissions separately- or at least i cant think of a case where a sit emight want the organism to be viewable, but not the analysis, *plus* they've attached this field and want it to only display for some users.

Reply:
Agreed. I'll remove the check.